ui/src/cohortsSlice.ts

```diff
@@ -36,18 +36,43 @@ const cohortsSlice = createSlice({
           cohort.groups.push(action.payload.group);
         }
       },
-      prepare: (cohortId: string, kind: GroupKind, criteria: Criteria) => ({
+      prepare: (
+        cohortId: string,
+        name: string,
```

Comment:
No need to add this parameter since it always starts empty.

Reply:
done